examples/tv-casting-app/tv-casting-common/tv-casting-app.matter

```diff
@@ -782,19 +782,6 @@ server cluster IasZone = 1280 {
     kInvalidZoneType = 65535;
   }
 
-  bitmap IasZoneStatus : BITMAP16 {
-    kAlarm1 = 0x1;
-    kAlarm2 = 0x2;
-    kTamper = 0x4;
-    kBattery = 0x8;
-    kSupervisionReports = 0x10;
-    kRestoreReports = 0x20;
-    kTrouble = 0x40;
-    kAc = 0x80;
-    kTest = 0x100;
-    kBatteryDefect = 0x200;
-  }
-
   readonly attribute enum8 zoneState = 0;
```

Comment:
The defines seemed useful as they are part of bitmap16. Or are we saying that zoneStatus should have been 'IasZoneStatus' instead of bitmap16?

Reply:
I believe this was removed because of the change in types.xml where we had:
<bitmap name="IasZoneStatus" type="BITMAP16">
     <cluster code="0x0500"/>
     <cluster code="0x0501"/>
I remember running into an issue where clusters 0x0500 and 0x0501 did not exist for one of the use cases but the data type was referring to a cluster which did not exist. So I removed those 2 cluster references which may have led to this change. I can confirm again by testing this locally.